liquibase-standard/src/main/java/liquibase/changelog/visitor/UpdateVisitor.java

```diff
@@ -27,17 +31,26 @@ public class UpdateVisitor implements ChangeSetVisitor {
 
     private ChangeExecListener execListener;
 
+    private ShouldRunChangeSetFilter shouldRunChangeSetFilter;
+
     /**
      * @deprecated - please use the constructor with ChangeExecListener, which can be null.
      */
     @Deprecated
     public UpdateVisitor(Database database) {
         this.database = database;
     }
-    
+
+    @Deprecated
```

Comment:
Can you explain how this class should be instantiated with this newly deprecated constructor?

Reply:
Thinking a bit more, I believe we don't need to deprecate it. I added a validation for `shouldRunChangeSetFilter `so it will behave the same when this constructor is used.